book/src/articles/objects_deep_dive.md

```diff
@@ -0,0 +1,136 @@
+# Objects deep dive
+
+An object is a sprite drawn to an arbitrary part of the screen.
+They are typically used for anything that moves such as characters and NPCs.
+All objects can be flipped and affine objects can have an affine transformation applied to them that can rotate and scale them.
+
+
+## Importing sprites
```

Comment:
I think it would be worth including a screenshot of aseprite showing the tags and how to add them (or linking to the aseprite documentation about adding tags?)